<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContainerLoadingCharges</class>
 <widget class="QMainWindow" name="ContainerLoadingCharges">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>775</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>972</width>
    <height>727</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>A22. Container Loading Charges Master Data Entry</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>950</width>
        <height>654</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>11</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignLeft">
           <widget class="QFrame" name="frame_4">
            <property name="minimumSize">
             <size>
              <width>946</width>
              <height>49</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>946</width>
              <height>49</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>500</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Customer ID:</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="customerID">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QFrame" name="frame_5">
            <property name="minimumSize">
             <size>
              <width>946</width>
              <height>49</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>946</width>
              <height>49</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>228</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Container Charges:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="containerCharges">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_11">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignLeft">
              <widget class="QLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Handling Type:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignLeft">
              <widget class="QRadioButton" name="tonnageBtn">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Tonnage</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="tripBtn">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Trip</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>500</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignLeft">
           <widget class="QFrame" name="frame_6">
            <property name="minimumSize">
             <size>
              <width>729</width>
              <height>555</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignBottom">
              <widget class="QFrame" name="frame_8">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>70</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="leftMargin">
                 <number>11</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>11</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item alignment="Qt::AlignBottom">
                 <widget class="QLabel" name="label_3">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Find</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignBottom">
                 <widget class="QLineEdit" name="searchBar"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_9">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QTableView" name="tableView"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_7">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QFrame" name="frame_10">
               <property name="minimumSize">
                <size>
                 <width>193</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <widget class="QPushButton" name="saveButton">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>250</y>
                  <width>93</width>
                  <height>28</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
               <widget class="QPushButton" name="addButton">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>210</y>
                  <width>93</width>
                  <height>28</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
               <widget class="QPushButton" name="close">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>290</y>
                  <width>93</width>
                  <height>28</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Close</string>
                </property>
               </widget>
               <widget class="QPushButton" name="deleteButton">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>170</y>
                  <width>93</width>
                  <height>28</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
               <widget class="QPushButton" name="resetButton">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>130</y>
                  <width>93</width>
                  <height>28</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
